### DISCUSSION TITLE
problem converting lammps_type to mace_type

### DISCUSSION THREAD

**Discussion by phys-chem** [ACEsuit/mace#232] [Q&A]
I am running lammps cpu veriosn for MACE. However, I meet this error：
problem converting lammps_type to mace_type

 When I run create_lammps_model.py，I meet this warning: 
UserWarning: The TorchScript type system doesn't support instance-level annotations on empty non-base types in `__init__`. Instead, either 1) use a type annotation in the class body, or 2) wrap the type in `torch.jit.Attribute`. warnings.warn("The TorchScript type system doesn't support "

**Comment by wcwitt** [comment]
Thanks - that's just a warning and I don't believe it's a problem. You still get an output model that works, right?

**Reply by phys-chem** [reply]
Thank you! It does not work. I also meet a memory problem: 

> RuntimeError: CUDA out of memory. Tried to allocate 3.44 GiB (GPU 0; 10.92 GiB total capacity; 6.96 GiB already allocated; 3.36 GiB free; 7.01 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

Is it caused by the environment? Thank you very much!

**Comment by wcwitt** [comment]
I'm not sure - but It looks like you might just need more memory. That's happening during the conversion stage?

**Comment by phys-chem** [comment]
> I'm not sure - but It looks like you might just need more memory. That's happening during the conversion stage?

Thank you! I think there is some problems in my device. It happens during running LAMMPS.
